test/services/mapping/declaration/DeclarantBuilderSpec.scala

```diff
@@ -55,5 +58,61 @@ class DeclarantBuilderSpec extends WordSpec with Matchers with MockitoSugar {
         declarant.getName should be(null)
       }
     }
+
+    "build then add" when {
+      "no declarant details" in {
+        val model = aCacheModel(withoutDeclarantDetails())
+        val declaration = new Declaration()
+
+        builder.buildThenAdd(model, declaration)
+
+        declaration.getDeclarant should be(null)
+      }
+
+      "no eori" in {
+        val model = aCacheModel(withDeclarantDetails(eori = None, address = Some(Address("name", "line", "city", "postcode", "United Kingdom"))))
+        val declaration = new Declaration()
+
+        builder.buildThenAdd(model, declaration)
+
+        declaration.getDeclarant.getID should be(null)
+      }
+
+      "no address" in {
+        val model = aCacheModel(withDeclarantDetails(eori = Some("eori"), address = None))
+        val declaration = new Declaration()
+
+        builder.buildThenAdd(model, declaration)
+
+        declaration.getDeclarant.getAddress should be(null)
+      }
+
+      "unknown country" in {
+        val model = aCacheModel(withDeclarantDetails(eori = Some("eori"), address = Some(Address("name", "line", "city", "postcode", "unknown"))))
+        val declaration = new Declaration()
+
+        builder.buildThenAdd(model, declaration)
+
+        declaration.getDeclarant.getAddress.getCountryCode.getValue should be("")
+      }
+
+      "populated" in {
+        val model = aCacheModel(withDeclarantDetails(eori = Some("eori"), address = Some(Address("name", "line", "city", "postcode", "United Kingdom"))))
+        val declaration = new Declaration()
+
+        builder.buildThenAdd(model, declaration)
+
+        declaration.getDeclarant.getAddress.getLine.getValue should be("line")
+        declaration.getDeclarant.getAddress.getCityName.getValue should be("city")
+        declaration.getDeclarant.getAddress.getPostcodeID.getValue should be("postcode")
+        declaration.getDeclarant.getAddress.getCountryCode.getValue should be("GB")
+        declaration.getDeclarant.getName.getValue should be("name")
+        declaration.getDeclarant.getID.getValue should be("eori")
+      }
+    }
+  }
+
+  private def builder = {
```

Comment:
You can just make it one line without parenthesis. Just style comment to comment something :)